src/focus.service.ts

```diff
@@ -508,7 +508,7 @@ export class FocusService {
    */
   private isFocusable(el: Element) {
     const role = el.getAttribute('role');
-    const tabIndex = el.getAttribute('tabIndex');
+    const tabIndex = el.getAttribute('tabindex');
```

Comment:
Actually I changed `root` to a `HTMLElement` already. According to http://stackoverflow.com/questions/38234810/get-htmlelement-from-element we can cast it if we're sure it's returning `HTMLElement`s. I'll just cast them then